src/libostree/libostree-released.sym

```diff
@@ -687,6 +687,7 @@ LIBOSTREE_2022.5 {
 global:
   ostree_kernel_args_append_if_missing;
   ostree_sysroot_deployment_set_kargs_in_place;
+  ostree_kernel_args_contains;
```

Comment:
This is wrong since 2022.5 is already released. You'll need to add a new section for `LIBOSTREE_2022.7` that starts from `LIBOSTREE_2022.5`. And actually that should be in `libostree-devel.sym` as noted in the comment below this.